packages/charts/react-charting/src/components/VerticalBarChart/VerticalBarChartRTL.test.tsx

```diff
@@ -652,6 +652,84 @@ describe('Vertical bar chart - Subcomponent Legends', () => {
       expect(bars[7]).toHaveStyle('opacity: 0.1');
     },
   );
+
+  testWithWait(
+    'Should reduce the opacity of the other bars/lines and their legends on mouse over multiple legends',
+    VerticalBarChart,
+    { data: pointsWithLine, lineLegendText: 'just line', legendProps: { canSelectMultipleLegends: true } },
+    container => {
+      const bars = getById(container, /_VBC_bar/i);
+      const line = getById(container, /_VBC_line/i)[0];
+      const legends = screen.getAllByText((content, element) => element!.tagName.toLowerCase() === 'button');
+      expect(line).toBeDefined();
+      expect(bars).toHaveLength(8);
+      expect(legends).toHaveLength(9);
+      fireEvent.click(screen.getByText('just line'));
+      fireEvent.click(screen.getByText('Oranges'));
+      expect(line.getAttribute('opacity')).toEqual('1');
+      expect(screen.getByText('Dogs')).toHaveStyle('opacity: 0.67');
```

Comment:
>toHaveStyle [](http://example.com/codeflow?start=39&length=11)

you are not checking for oranges opacity

Reply:
added